include/ignition/gui/GuiEvents.hh

```diff
@@ -255,6 +258,59 @@ namespace ignition
         /// for this event.
         private: bool menuEnabled;
       };
+
+      /// \brief Event which is called to broadcast the key release within
+      /// the scene.
+      class KeyRelease : public QEvent
```

Comment:
Can these have the `OnScene` suffix like on #228 for clarity?

Reply:
Done https://github.com/ignitionrobotics/ign-gui/pull/224/commits/906b0dd84176f35c9627b5140a426c86b2e86573